developer/services/qrexec3.md

```diff
@@ -32,14 +32,14 @@ It allows users and developers to use and design secure inter-VM tools.
 
 ## Qrexec basics ##
 
-Qrexec is built on top of vchan (a library providing data links between VMs).
-During domain creation a process named `qrexec-daemon` is started in dom0, and a process named `qrexec-agent` is started in the VM.
-They are connected over **vchan** channel.
-`qrexec-daemon` listens for connections from dom0 utility named `qrexec-client`.
-Typically, the first thing that a `qrexec-client` instance does is to send a request to `qrexec-daemon` to start a process (let's name it `VMprocess`) with a given command line in a specified VM (`someVM`).
-`qrexec-daemon` assigns unique vchan connection details and sends them both to `qrexec-client` (in dom0) and `qrexec-agent` (in `someVM`).
-`qrexec-client` starts a vchan server which `qrexec-agent` connects to.
-Since then, stdin/stdout/stderr from the VMprocess is passed via vchan between `qrexec-agent` and the `qrexec-client` process.
+Qrexec is built on top of *vchan*, a Xen library providing data links between VMs.
+During domain creation, a process named `qrexec-daemon` is started in dom0, and a process named `qrexec-agent` is started in the VM.
+They are connected over a **vchan** channel.
+`qrexec-daemon` listens for connections from a dom0 utility named `qrexec-client`.
+Typically, the first thing that a `qrexec-client` instance does is to send a request to the `qrexec-agent` running in a specified VM (`someVM`) to start a process (let's name it `VMprocess`)
```

Comment:
Does the request go to `qrexec-daemon` or `qrexec-agent`? My understand so far is that qrexec-daemon is only present in dom0.

Reply:
All control messages from `qrexec-client` go to `qrexec-daemon`, which then either rely them to `qrexec-agent` or handle itself. So, it should be "send a request to the `qrexec-daemon` which rely it further to `qrexec-agent`".